--- liboping-1.10.0/src/Makefile.in.orig	2019-04-09 08:49:58.470387572 +0200
+++ liboping-1.10.0/src/Makefile.in	2019-04-09 08:50:08.080696809 +0200
@@ -387,7 +387,7 @@
 top_srcdir = @top_srcdir@
 AUTOMAKE_OPTIONS = foreign no-dependencies
 SUBDIRS = mans
-@COMPILER_IS_GCC_TRUE@AM_CFLAGS = -Wall -Werror
+@COMPILER_IS_GCC_TRUE@AM_CFLAGS = -Wall
 include_HEADERS = oping.h
 lib_LTLIBRARIES = liboping.la
 
